addons-cluster/postgresql/templates/rbac.yaml

```diff
@@ -1 +1,37 @@
-{{- include "kblib.rbac" . }}
\ No newline at end of file
+{{- include "kblib.rbac" . }}
+---
+{{- if .Values.etcd.proxyEnabled }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  namespace: {{ .Release.Namespace }}
+  name: {{ printf "%s-etcd-clean-role" (include "kblib.clusterName" .) }}
```

Comment:
Introducing a new role in Helm to implement this functionality is inappropriate because the lifecycle of this role is tied to Helm rather than the cluster's lifecycle. This poses significant challenges for subsequent maintenance and management.

A correct approach is that if a component requires additional permissions, it should be explicitly declared before it can be applied in scenarios such as lifecycle actions. The management of these permissions should be part of the controller's functionality. You can consider extending the implementation of the current PolicyRules (not yet implemented) in cmpd to meet such requirements.